model/container/registry.yaml

```diff
@@ -19,12 +19,26 @@ groups:
           [identify Docker containers](https://docs.docker.com/engine/containers/run/#container-identification).
           The UUID might be abbreviated.
         examples: ['a3bf90e006b2']
-      - id: container.runtime
+      - id: container.runtime.name
```

Comment:
Could you add this in the [non-normative migration guidance](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/non-normative/k8s-migration.md#summary-of-changes)? It affects the [k8sclusterreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/9f1f674d7af62267f641c0b631d1326815ff80d1/receiver/k8sclusterreceiver/metadata.yaml#L180-L188) and we try to keep track of such changes to prepare the users.